app/root.tsx

```diff
@@ -77,7 +84,7 @@ export async function loader({ request }: Route.LoaderArgs) {
     cookieSettings: cookieSettings || null,
     userAgentString: userAgentString || '',
     origin: url.origin,
-    domain: url.origin.replace('https://', '').replace('http://', ''),
+    domain: stripProtocolFromUrl(url.origin),
```

Comment:
I think you can just do url.host and you will get what you want without this helper method